examples/birthday_dividends.py

```diff
@@ -41,7 +40,7 @@ def make_webdriver():
 for s in (sp500[:250], sp500[250:]):
     r = client.search_instruments(
         s, tda.client.Client.Instrument.Projection.FUNDAMENTAL)
-    assert r.ok, r.raise_for_status()
+    assert r.status_code == 200, r.raise_for_status()
```

Comment:
what is wrong with ok?

Reply:
In the new HTTPX library, there is no "ok" property. That's a requests-specific thing.